<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="MyGlWidget" name="myGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1284</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_as"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>248</width>
     <height>273</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="toolsTab">
        <attribute name="title">
         <string>Tools</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTreeView" name="treeView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="paramTabs">
        <attribute name="title">
         <string>Param</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <property name="horizontalSpacing">
             <number>1</number>
            </property>
            <property name="verticalSpacing">
             <number>4</number>
            </property>
            <item row="1" column="1">
             <widget class="QLineEdit" name="widthPanel">
              <property name="text">
               <string>0.5</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="heightPanel">
              <property name="text">
               <string>0.5</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Длинна</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="topPanel">
              <property name="text">
               <string>0.2</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Толщина</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Ширина</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTools"/>
   <addaction name="actionVerticalDrill"/>
   <addaction name="actionHorizontalDrill"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="features">
    <set>QDockWidget::DockWidgetFeatureMask</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>2</number>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="2" colspan="2">
               <widget class="QLineEdit" name="lineX"/>
              </item>
              <item row="1" column="2" colspan="3">
               <widget class="QLineEdit" name="lineDiam"/>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">color: rgb(85, 170, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
                </property>
                <property name="text">
                 <string>X    Y</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2" colspan="3">
               <widget class="QLineEdit" name="lineDepth"/>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QLabel" name="label_5">
                <property name="styleSheet">
                 <string notr="true">color: rgb(85, 170, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
                </property>
                <property name="text">
                 <string>Diametr</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4" alignment="Qt::AlignHCenter">
               <widget class="QLineEdit" name="lineY"/>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true">color: rgb(85, 170, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
                </property>
                <property name="text">
                 <string>Depth</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>2</number>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>2</number>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/icons/document-new.svg</normaloff>:/new/icons/icons/document-new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/icons/document-save.svg</normaloff>:/new/icons/icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/icons/document-open.svg</normaloff>:/new/icons/icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save_as">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/icons/document-save.svg</normaloff>:/new/icons/icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/icons/sverlo1.svg</normaloff>:/new/icons/icons/sverlo1.svg</iconset>
   </property>
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionVerticalDrill">
   <property name="icon">
    <iconset>
     <normalon>:/new/icons/icons/vertical.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>VerticalDrill</string>
   </property>
  </action>
  <action name="actionHorizontalDrill">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/icons/horixontal.svg</normaloff>
     <normalon>:/new/icons/icons/vertical.svg</normalon>:/new/icons/icons/horixontal.svg</iconset>
   </property>
   <property name="text">
    <string>HorizontalDrill</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGlWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>myglwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>heightPanel</tabstop>
  <tabstop>widthPanel</tabstop>
  <tabstop>topPanel</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
